index.html

```diff
@@ -293,62 +293,70 @@
+viable degree of oversight and accountability such that technical and procedural means to guarantee
+the maintenance of pseudonymity are preserved.
+
+This is more difficult when the [=pseudonymous data=] is shared between several [=parties=]. In such
+cases, typical controls would include making sure that
 
-* the identifiers used in the data are under the direct and exclusive control of
-  the [=first party=]; and
+* the [=identifiers=] used in the [=data=] are under the direct and exclusive control of
+  the [=first party=];
 
-* when these identifiers are shared with a [=third party=], they are made unique
+* when these [=identifiers=] are shared with a [=third party=], they are made unique
   to that [=third party=] such that if they are shared with more than one
-  [=third party=] these cannot then match them up with one another; and
+  [=third party=] these cannot then match them up with one another;
 
-* there is a strong level of confidence that no [=third party=] can match them
-  to any data other than that obtained through interactions with the [=first
-  party=]; and
+* there is a strong level of confidence that no [=third party=] can match the [=data=]
+  with any data other than that obtained through interactions with the [=first
+  party=];
 
-* any [=third party=] receiving such identifiers is barred (eg. based on legal
-  terms) from sharing them or the related data further; and
+* any [=third party=] receiving such [=data=] is barred (eg. based on legal terms)
+  from sharing it further;
 
 * technical measures exist to prevent re-identification or the joining of
-  different data sets involving these identifiers, notably against timing or
+  different data sets involving this [=data=], notably against timing or
   k-anonymity attacks; and
 
 * there exist contractual terms between the [=first party=] and [=third party=]
   describing the limited [=purpose=] for which the data is being shared.
 
-This can ensure that [=pseudonymous data=] is used in a manner that provides a minimum
-degree of governance such that technical and procedural means to guarantee the
-maintenance of pseudonymity are preserved. Note that [=pseudonymity=], on its own, is not
-sufficient to render [=data processing=] [=appropriate=].
+Note that [=pseudonymity=], on its own, is not sufficient to render [=data processing=]
+[=appropriate=].
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who, at least in the [=context=] of
-the [=processing=] being discussed, are unable to exercise sufficient
-self-determination for any consent they may provide to be receivable. This includes for
-example children, employees with respect to their employers, people in some situations of
-intellectual or psychological impairment, or refugees.
+A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable
+to exercise sufficient self-determination in the [=context=] of a specific kind of
```

Comment:
"Sufficient self-determination" to what? What about "may be more easily coerced or tricked than an average member of the population"?